md/BonitaBPM_platform_setup.md

```diff
@@ -102,27 +109,69 @@ If you want to modify the configuration of an already initialized Bonita BPM Pla
 Note that the *Platform Setup tool* does not need the Bonita BPM Server to be running for the configuration to be updated. However, the server needs to be restarted for the changes to take effect.
```

Comment:
Ok for the message in warning instead of info. But the sentence seems clear enough for me. I would keep it like that.